.github/workflows/pr_infra.yaml

```diff
@@ -0,0 +1,26 @@
+name: (PR) Infra WestEurope
```

Comment:
can we use another region?

Reply:
Not yet, but probably we need to start to think about it and eventually have a discussion about common resources like vnets, apim and app gateway
I'd keep fims in we for a while, but we need to have the discussion before going live